deploy/README.md

```diff
@@ -176,6 +176,20 @@ This way, `topolvm-scheduler` is exposed by LoadBalancer service.
 
 Then edit `urlPrefix` in [./scheduler-config/scheduler-policy.cfg](./scheduler-config/scheduler-policy.cfg) to specify the LoadBalancer address.
 
+### Scoring tuning
+
+The prioritize of Pod scheduling by `topolvm-scheduler` can be tuned via the following two ways:
+1. Adjust the divisor parameter in the scoring expression
+2. Change the weight for the scoring by `topolvm-scheduler` against the default by kube-scheduler
```

Comment:
"scoring by `topolvm-scheduler`" seems verbose. I consider "Weight  for the scoring" is enough. This indication applies to the same expressions in the following descriptions.

Reply:
I think so about the first one, but about the latter one,
>So if you want to give a priority to the scoring by `topolvm-scheduler`,

I think `by topolvm-scheduler` seems to be needed to specify the object.